pkg/providers/discoverers/cloud_foundry/test_data/complete-manifest/manifest.yml

```
@@ -0,0 +1,54 @@
+# taken from https://v3-apidocs.cloudfoundry.org/version/3.76.0/index.html#the-app-manifest-specification
+# added application's name 'name: complete' since it is missing in the example and it
+# is required by the spec: https://docs.cloudfoundry.org/devguide/deploy-apps/manifest-attributes.html#minimal-manifest
+---
+applications:
+- name: complete
+  buildpacks:
+    - ruby_buildpack
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 2 but found 4

(indentation)